tests/Exclusion/Check/ExcludeByDocBlockTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Tests\Exclusion\Check;
+
+use Iterator;
+use Rector\DeadCode\Rector\ClassMethod\RemoveEmptyClassMethodRector;
+use Rector\DeadCode\Rector\Plus\RemoveZeroAndOneBinaryRector;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+final class ExcludeByDocBlockTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideDataForTest()
+     */
+    public function test(string $file): void
+    {
+        $this->doTestFile($file);
+    }
+
+    public function provideDataForTest(): Iterator
+    {
+        yield [__DIR__ . '/Fixture/fixture.php.inc'];
+        yield [__DIR__ . '/Fixture/fixture-norector.php.inc'];
+        yield [__DIR__ . '/Fixture/fixture2.php.inc'];
+        yield [__DIR__ . '/Fixture/fixture2-norector.php.inc'];
+        yield [__DIR__ . '/Fixture/fixture3.php.inc'];
```

Comment:
Eplicit fixture naming would be better for further maintenance, e.g. "exclude_with_rule_name.php.inc" etc.

Reply:
fixed